packages/yourloops/components/chat/chat-widget.tsx

```diff
@@ -219,11 +218,11 @@ function ChatWidget(props: ChatWidgetProps): JSX.Element {
       }
       <div id="chat-widget-footer" className={classes.chatWidgetFooter}>
         <div className={classes.chatWidgetHCPToggle}>
-          { patientId !== userId &&
+          { userRole === "hcp" &&
```

Comment:
Can't you use something like:
`userRole === UserRoles.hcp`?
Also, I think that this file needs some code reformat

Reply:
It's already the case ... maybe you did the review on outdated code ?

Reply:
done, code pushed right now ...